wren-tests/src/test/java/io/wren/testing/duckdb/TestWrenWithDuckDBTableFunction.java

```diff
@@ -33,6 +33,7 @@
 import static java.util.Objects.requireNonNull;
 import static org.assertj.core.api.Assertions.assertThat;
 
+@Test(singleThreaded = true)
```

Comment:
Note: Reloading DuckDB would be triggered by other test cases (set init-sql). Run tests sequentially to avoid testing failure.

Reply:
Another test class, `TestDuckDBResource`, which uses `setDuckDBInitSQL`, is already set to `singleThreaded`. They don't affect each other. We don't need to make any adjustments now. If we encounter any issues in the future, we can then modify the test suite as necessary.